### PR TITLE
GHC-132: Add Covid Vaccine to Vaccination form

### PR DESCRIPTION
**Pull request by basrem**

### CODE REVIEW

**Comment by reagan-meant** [comment]
Make these [changes](https://github.com/globalhealthcoalition/openmrs-config-ghc/pull/90) here

File: configuration/concepts/vaccination.csv [hunk 1]
```diff
@@ -14,4 +14,6 @@ dd07efeb-47f1-4b72-9584-0152837b1b3b,,,HPV,VPH,HPV,VPH,,Misc,Text,,,,,,,,,,,
 43c8df3c-6c4c-4921-ba7c-a7d0f9dba4db,,,Meningococcal,Meningocócica,Meningococcal,Meningocócica,,Misc,Text,,,,,,,,,,,
 a415ac86-04d5-4e7b-a19b-c4abd431d930,,,HepB,HepB,HepB,HepB,,Misc,Text,,,,,,,,,,,
 018bc554-4d36-4887-a387-fb1b8bb20216,,,Influenza,Influenza,Influenza,Influenza,,Misc,Text,,,,,,,,,,,
-3c25e650-ccfa-48da-b276-a7872796b113,,,Vaccination History,Historial de vacunación,Vaccination History,Historial de vacunación,Vaccination History,ConvSet,N/A,,,Rotavirus; dT; Rougeole/Rubeole; Pentavalent; Polio oral; Pneumocoque; BCG; MMR; Var; HepA; Tdap; HPV; Meningococcal; HepB; Influenza,,,,,,,,
\ No newline at end of file
+2258b601-0c96-44c7-adb9-9d14d290da62,,,Covid,COVID-19,Covid,COVID-19,,Misc,Text,,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/vaccination.csv]: Use the same casing as "COVID-19" instead of Covid

File: configuration/concepts/vaccination.csv [hunk 2]
```diff
@@ -14,4 +14,6 @@ dd07efeb-47f1-4b72-9584-0152837b1b3b,,,HPV,VPH,HPV,VPH,,Misc,Text,,,,,,,,,,,
...
+f109a030-ea70-4884-b947-63d5ba25f2cf,,,Vaccination Document,documento de vacunación,Vaccination Document,documento de vacunación,,Image,Complex,ImageUrlHandler,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/vaccination.csv]: Use "Documento de vacunación" for both short and full name

File: configuration/bahmniforms/vaccination_history.json [hunk 3]
```diff
@@ -1,103 +1,137 @@
...
-            {
-              "rel": "self",
-              "uri": "http://openmrs:8080/openmrs/ws/rest/v1/user/b3167dfe-0adb-11ec-a6ca-0242c0a83003"
-            }
-          ]
-        },
-        "dateChanged": "2021-09-01T05:10:56.000+0000"
+  "formJson": {
+    "id": 16,
+    "uuid": "248b4750-01f5-4052-81bb-d180712defdf",
```
Comment (reagan-meant) [inline configuration/bahmniforms/vaccination_history.json]: We agreed to keep the same uuids

**Review by reagan-meant — CHANGES_REQUESTED** [review]

File: configuration/concepts/vaccination.csv (hunk 1 above)
Comment (basrem) [reply configuration/concepts/vaccination.csv]: Done

File: configuration/concepts/vaccination.csv (hunk 2 above)
Comment (basrem) [reply configuration/concepts/vaccination.csv]: Done

File: configuration/bahmniforms/vaccination_history.json (hunk 3 above)
Comment (basrem) [reply configuration/bahmniforms/vaccination_history.json]: Noted

File: configuration/concepts/vaccination.csv [hunk 4]
```diff
@@ -14,4 +14,6 @@ dd07efeb-47f1-4b72-9584-0152837b1b3b,,,HPV,VPH,HPV,VPH,,Misc,Text,,,,,,,,,,,
 43c8df3c-6c4c-4921-ba7c-a7d0f9dba4db,,,Meningococcal,Meningocócica,Meningococcal,Meningocócica,,Misc,Text,,,,,,,,,,,
 a415ac86-04d5-4e7b-a19b-c4abd431d930,,,HepB,HepB,HepB,HepB,,Misc,Text,,,,,,,,,,,
 018bc554-4d36-4887-a387-fb1b8bb20216,,,Influenza,Influenza,Influenza,Influenza,,Misc,Text,,,,,,,,,,,
-3c25e650-ccfa-48da-b276-a7872796b113,,,Vaccination History,Historial de vacunación,Vaccination History,Historial de vacunación,Vaccination History,ConvSet,N/A,,,Rotavirus; dT; Rougeole/Rubeole; Pentavalent; Polio oral; Pneumocoque; BCG; MMR; Var; HepA; Tdap; HPV; Meningococcal; HepB; Influenza,,,,,,,,
\ No newline at end of file
+2258b601-0c96-44c7-adb9-9d14d290da62,,,COVID-19,COVID-19,COVID-19,COVID-19,,Misc,Text,,,,,,,,,,,
+f109a030-ea70-4884-b947-63d5ba25f2cf,,,Vaccination Document,Documento de vacunación,Vaccination Document,D 1ocumento de vacunación,,Image,Complex,ImageUrlHandler,,,,,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/vaccination.csv]: is` D 1ocumento de vacunación `expected?

**Review by reagan-meant — CHANGES_REQUESTED** [review]

**Comment by reagan-meant** [comment]
Closing this PR in preference to [this](https://github.com/globalhealthcoalition/openmrs-config-ghc/pull/96)

**Pull request closed without merging**